crates/wash-cli/tests/common/mod.rs

```diff
@@ -250,6 +250,8 @@ impl TestWashInstance {
         assert!(status.success());
 
         let out = read_to_string(&log_path).context("could not read output of wash up")?;
+        println!("wash up output: {}", out);
+        eprintln!("wash up output: {}", out);
```

Comment:
Merrrrrr

Reply:
Yeah this is from my temporary testing things, found out the test is failing on an `etxtfilebusy` on downloading wadm 0.9.1? I assume we're getting rate limited so I'm gonna re-run tomorrow.